aiida/backends/manager.py

```diff
@@ -191,11 +191,8 @@ def get_schema_version_reset(self, schema_generation_code):
         """
 
     @abc.abstractmethod
-    def get_schema_version_database(self):
-        """Return the database schema version.
-
-        :return: `distutils.version.LooseVersion` with schema version of the database
-        """
+    def get_schema_version_backend(self):
+        """Return the current schema version."""
```

Comment:
Same thing here, would be good to give a bit more detail explaining that this is the version that is stored in the currently configured backend.

Reply:
done